CorePayments/src/test/java/com/paypal/android/corepayments/APIUnitTest.kt

```diff
@@ -11,8 +11,7 @@ import io.mockk.slot
 import kotlinx.coroutines.ExperimentalCoroutinesApi
 import kotlinx.coroutines.test.runTest
 import org.json.JSONObject
-import org.junit.Assert.assertEquals
-import org.junit.Assert.assertSame
+import org.junit.Assert.*
```

Comment:
Basic Q again, I assume the `*` just imports all of `junit.Assert`? If so that's cool and I can see this being useful since some files have what feels like 40 imports. Are these imports generated automagically like if we use X amount from an import it switches to use the `*` or is it a manual process?

Reply:
Yeh, the * imports everything within the namespace. We have [this Detekt rule](https://github.com/paypal/Android-SDK/blob/0d46271a16fc3193db01d5f8fa0c33bd1fae13f5/detekt/detekt-config.yml#L786) that does not allow wildcard imports in source files, but does allow them in test files. I am guessing we added it to avoid importing unnecessary classes in the SDK source itself.

The import statements were generated automatically by Android Studio (command + enter on a red class name usually adds the import statement for you). Some of them defaulted to * in the source file, which I had to change to specify the exact class, in order to satisfy the linter.

Reply:
Gotcha - that's cool, TIL!